src/components/Form.js

```diff
@@ -76,12 +86,30 @@ const StyledLabel = styled.label`
   flex-direction: column;
   flex-wrap: wrap;
   margin-bottom: 20px;
+  width: 100%;
+
+  ${minBreakpointQuery.smedium`
+    &:nth-child(1) {
```

Comment:
None of these grid areas will be needed as the grid styles on the form will naturally lay them out, the only thing you will needed is `grid-column: span 2;` on two of them so they go full width, you can do that with a prop similar to how they work on the Container ui component, or a better example with an if statement is on the Accordion or Tabs component on the base repo